app/Http/Controllers/LoginController.php

```diff
@@ -16,14 +16,21 @@ public function getLogin()
 
     public function postLogin(LoginRequest $request)
     {
-        $credentials = array(
+        $is_admin = array(
```

Comment:
```suggestion
        $is_admin = [];
```
chú ý cách khởi tạo array